components/tools/OmeroWeb/omeroweb/feedback/views.py

```diff
@@ -131,12 +130,13 @@ def send_comment(request):
 ##############################################################################
 # handlers
 
-
 def csrf_failure(request, reason=""):
-    logger.warn('csrf_failure: Forbidden')
-    t = template_loader.get_template("403_csrf.html")
-    c = RequestContext(request, {})
-    return HttpResponseForbidden(t.render(c))
+    """
+    Always return Json response
+    since this is accepted by browser and API users
+    """
+    error = "CSRF Error. You need to include 'X-CSRFToken' in header"
```

Comment:
I would just say:


_CSRF Error.  You have to include valid CSRF tokens for any POST, PUT, PATCH or DELETE operations. In order to make AJAX requests, you need to include CSRF token in the HTTP header, for more details check https://docs.djangoproject.com/en/1.8/ref/csrf/_